apps/backend/src/libs/modules/tokenizer/tokenizer.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseTokenizer } from "./base-tokenizer.module.js";
+
+const tokenizer = new BaseTokenizer();
```

Comment:
```suggestion
const token = new BaseToken();
```
do not forget about all file names

Reply:
> do not forget about all file names
of course!)